### PR TITLE
Remove automatic language identification

### PR DESCRIPTION
**Pull request by oliverkinch**
Use hardcoded mapping from newsroom to language instead of doing language identification, as suggested in https://github.com/ScandEval/ScandEval/issues/526#issuecomment-2451747472.

Fix https://github.com/ScandEval/ScandEval/issues/526

### CODE REVIEW

File: src/scripts/create_schibsted.py [hunk 1]
```diff
@@ -55,35 +74,9 @@ def main():
...
-            return "no"
-        return "other"
-
-    dataset_sv = dataset.filter(
-        lambda x: detect_language(text=x["article_title"]) == "sv"
-    )
-    dataset_no = dataset.filter(
-        lambda x: detect_language(text=x["article_title"]) == "no"
-    )
+    dataset_sv = dataset.filter(lambda x: x["language"] == "se")
```
Comment (saattrupdan) [inline src/scripts/create_schibsted.py]: `se` is Sweden the country, not Swedish the language - the language is `sv`.

```suggestion
    dataset_sv = dataset.filter(lambda x: x["language"] == "sv")
```

File: src/scripts/create_schibsted.py [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
+    "dinepenger": "no",
+    "vgpartnerstudio": "no",
+    "bt": "no",
+    "tekno": "no",
+    "vg": "no",
+    "ap": "no",
+    "randaberg24": "no",
+    "ab": "se",
+    "sa": "no",
+}
```
Comment (saattrupdan) [inline src/scripts/create_schibsted.py]: `se` is Sweden the country, not Swedish the language - the language is `sv`.

```suggestion
NEWSROOM_TO_LANGUAGE = {
    "sno-commercial": "no",
    "vektklubb": "no",
    "e24": "no",
    "e24partnerstudio": "no",
    "dinepenger": "no",
    "vgpartnerstudio": "no",
    "bt": "no",
    "tekno": "no",
    "vg": "no",
    "ap": "no",
    "randaberg24": "no",
    "ab": "sv",
    "sa": "no",
}
```

**Review by saattrupdan — APPROVED** [review]
Small typo, otherwise looks good 🙂

**Comment by oliverkinch** [comment]
Fixes https://github.com/ScandEval/ScandEval/issues/526

**Pull request merged**